INSTALL.md

```diff
@@ -14,4 +14,75 @@ You can run tests with:
 cargo test
 ```
 
-Please refer to the cargo documentation for more detailed instructions.
\ No newline at end of file
+Please refer to the cargo documentation for more detailed instructions.
+
+
+### If you want to create a system service to run your application follow the instructions below:
+
+
+# Systemd setup for backend
+
+
+#### Create a simple file copying the contents below to the file, and use the command:
```

Comment:
Using you're preferred text editor, create a file in the `systemd` folder. For instance:

`sudo nano /etc/systemd/system/teosd.service`

Reply:
All formatting and wording comments have been modified.